.devcontainer/devcontainer.json

```
@@ -4,7 +4,7 @@
     "dockerfile": "Dockerfile",
     "context": ".."
   },
-  "appPort": 8123,
+  "appPort": 8124,
```

```suggestion
  "appPort":  "8124:8123",
```
Sorry, was a bit trigger happy, this need a mao from ext -> int